src/sentry/db/postgres/schema.py

```diff
@@ -130,7 +140,7 @@ def safe(self, safe):
     def schema_editor(self):
         if self._schema_editor is None:
             schema_editor_cls = (
-                SafePostgresDatabaseSchemaEditor if self.safe else PostgresDatabaseSchemaEditor
+                SafePostgresDatabaseSchemaEditor if self.safe else MakeBtreeGistSchemaEditor
```

Comment:
I'm not totally sure about doing this, because `self.safe` is set to false when `checked = False` is set on the a migration as well. Probably it doesn't hurt at all, because worse case all it does is add another `CREATE EXTENSION IF NOT EXISTS...`. I guess it's fine, maybe leave a comment explaining why this is here?

Reply:
I think that case would be exceedingly rare or probably would never happen -- we'd have to have a `checked = False` migration that also creates a table with this particular constraint type -- which we've only created this constraint type twice in sentry's history

and even then it'd just run a noop `create extension`